sim/core/statweight.go

```diff
@@ -163,7 +166,21 @@ func CalcStatWeight(swr proto.StatWeightsRequest, statsToWeigh []stats.Stat, ref
 		}
 	}
 
+	// Melee hit cap is 8% in WoTLK
+	melee2HHitCap := 8 * MeleeHitRatingPerHitChance
+	if swr.Debuffs != nil && swr.Debuffs.FaerieFire == proto.TristateEffect_TristateEffectImproved {
```

Comment:
I believe faerie fire is spell-only hit now

Reply:
You are correct, I'm gonna fix this